k2/csrc/aux_labels.h

```diff
@@ -50,28 +50,28 @@ struct AuxLabels {
 /*
   Maps auxiliary labels after an FSA operation where each arc in the output
   FSA corresponds to exactly one arc in the input FSA.
-     @param [in] labels_in   Labels on the arcs of the input FSA
+     @param [in] labels_in   int32_ts on the arcs of the input FSA
```

Comment:
the `int32_ts` means the plural of `int32_t`? (just a question as I really feel hard to get the meaning of `Auxint32_ts` from its name)